src/libs/EmojiTrie.ts

```diff
@@ -83,7 +101,7 @@ function createTrie(lang = CONST.LOCALES.DEFAULT) {
     return trie;
 }
 
-const emojiTrie = _.reduce(supportedLanguages, (prev, cur) => ({...prev, [cur]: createTrie(cur)}), {});
+const emojiTrie = supportedLanguages.reduce((prev, cur) => ({...prev, [cur]: createTrie(cur)}), {});
```

Comment:
Could you improve the return type of this variable? Currently is just `{}`.

<img width="761" alt="Screenshot 2023-09-18 at 17 37 00" src="https://github.com/Expensify/App/assets/20051562/ecf54581-f54a-4ad3-8ea9-4d18ad461c76">